src/main/java/org/candlepin/policy/js/RulesObjectMapper.java

```diff
@@ -62,29 +56,7 @@ public class RulesObjectMapper {
     @Inject
     @SuppressWarnings("checkstyle:indentation")
     public RulesObjectMapper() {
-        this.mapper = new ObjectMapper();
-
-        SimpleFilterProvider filterProvider = new SimpleFilterProvider()
-            .setFailOnUnknownId(false);
-
-        this.mapper.setFilterProvider(filterProvider);
-
-        Hibernate5Module hbm = new Hibernate5Module();
-        hbm.enable(Hibernate5Module.Feature.FORCE_LAZY_LOADING);
-
-        mapper.registerModule(new Jdk8Module());
-        mapper.registerModule(new JavaTimeModule());
-        mapper.registerModule(hbm);
-
-        // Very important for deployments so new rules files can return additional
-        // properties that this current server doesn't know how to serialize, but still
-        // shouldn't fail on.
-        this.mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-
-        AnnotationIntrospector primary = new JacksonAnnotationIntrospector();
-        AnnotationIntrospector secondary = new JaxbAnnotationIntrospector(mapper.getTypeFactory());
-        AnnotationIntrospector pair = new AnnotationIntrospectorPair(primary, secondary);
-        this.mapper.setAnnotationIntrospector(pair);
+        this.mapper = ObjectMapperFactory.getObjectMapper();
```

Comment:
If we're going to tie the object mapper to the DI framework in general, we should be doing that here as well.